database/seeders/VanityRedirectSeeder.php

```diff
@@ -11,6 +11,9 @@ class VanityRedirectSeeder extends Seeder
 {
     public function run(): void
     {
+        // XXX Without the generic pages seeded as well, this won't work as intended. I'm guessing there some vanity
+        // urls that are required to make the app function properly. Can we hardcode them here?
+
```

Comment:
Before we created a way for content creators to create Vanity Redirects in the site, there was an option within Generic Pages to have one Generic Page redirect to another. That was limited and hard for content folks to maintain ("how can I see all the redirects we've set up in the site?"). The work here that creates new Vanity Redirects based on Generic Page redirects was part of us automating the migration to the new functionality so content creators didn't have to move them one-by-one.

I don't think there are _any_ Vanity Redirects that HAVE to be in place for the app to work. I think the functionality in the seeder below can all be removed.